Automattic/newspack-theme
# Update theme screenshot

### All Submissions:


### Changes proposed in this Pull Request:

The theme's screenshot should not be Twenty Nineteen 😐 

There might be something we can do that's more representative of the theme's appearance, but for the short-term, let's at least switch it to the theme logo. 

Closes #272.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to WP Admin > Themes
3. Confirm the theme's screenshot is the Newspack logo:

![image](https://user-images.githubusercontent.com/177561/63482969-a159fb00-c44f-11e9-9d9a-35d3bdcaab30.png)

### Other information:
